docs/content.zh/docs/connectors/table/jdbc.md

```diff
@@ -672,12 +672,6 @@ Flink 支持连接到多个使用方言（dialect）的数据库，如 MySQL、O
       <td></td>
       <td><code>DECIMAL(20, 0)</code></td>
     </tr>
-    <tr>
-      <td><code>BIGINT</code></td>
-      <td></td>
-      <td><code>BIGINT</code></td>
-      <td><code>BIGINT</code></td>
-    </tr>
```

Comment:
Why is this removed?

Reply:
I think it's dumplicated.
![1671549417933](https://user-images.githubusercontent.com/13617900/208701065-d066c08e-9a42-4b9d-96ef-34174e791102.png)

Reply:
I don't think it's duplicated, because there's a BIGINT INT UNSIGNED / BIGINT UNSIGNED data type for MySQL (which are the first two lines), but there's also a regular `BIGINT` data type which exists for Postgres.

Reply:
I get it,Thanks for your patient reply.